mypyc/ops.py

```diff
@@ -544,6 +567,41 @@ def accept(self, visitor: 'OpVisitor[T]') -> T:
         return visitor.visit_load_int(self)
 
 
+class ModuleGet(RegisterOp):
+    """dest = name
+    """
+    def __init__(self, dest: Register, module: str) -> None:
+        self.dest = dest
+        self.module = module
+
+    def sources(self) -> List[Register]:
+        return []
+
+    def to_str(self, env: Environment) -> str:
+        return env.format('%r = module(%s)', self.dest, self.module)
```

Comment:
What about using `%r = module %s` as the format so that this looks different from a function call?